features/multi-column.yml

```diff
@@ -0,0 +1,19 @@
+name: Multi-column layout
+description: Multi-column layout flows an element's content across one or more vertical content areas in a single row, without affecting the `display` properties of its children.
+spec: https://drafts.csswg.org/css-multicol-1/
+group: layout
```

Comment:
How would you feel about a `multicolumn` group?

Reply:
I'm all for it, particularly since `multi-column` and `column-fill` won't be grouped alphabetically

Reply:
Do it! Make layout the parent, and add a commented out description (they want to support descriptions down the road and want to be prepared)